### ISSUE TITLE
Sub feed

### ISSUE THREAD

**Issue by Gamey404** [deep-gaurav/rusty_pipe#3]
Are there any plans on implementing some sort of subscription system (like invidious and newpipe)? 

(Hope that here is the right place to ask sorry if it isnt

**Comment by deep-gaurav** [comment]
Yes there will be a simple service worker and indexed db based local subscription system.

But it'll take a while as there are many things that need to be completed before it, like channel and playlists.
